tools/NuGet/BuildPackages.bat

```diff
@@ -1,23 +1,14 @@
-﻿:: Argument %1: path to template folder
```

Comment:
Looks like you rolled this file into a previous version, can you revert the change to this file only?